### ISSUE TITLE
Allow primitive properties to be passed as keyword arguments.

### ISSUE THREAD

**Issue by dcrawforAtAnsys** [ansys/pyaedt#4010]
### Description of the feature

Properties such as ``color`` and ``transparency`` should be passed as optional keywords when creating primitives.

### Steps for implementing the feature

```python
def create_rectangle(self, csPlane, position, dimension_list, name=None, matname=None, is_covered=True, **kwargs):

# code

if "color" in kwargs.keys():
   self.color = kwargs["color"]
```

### Useful links and references

I think this is a reasonable way to simplify syntax when a new primitive should have a specific color, transparency, etc. Rather than creating the primitive and then assigning a value to the property.

**Comment by Samuelopez-ansys** [comment]
@dcrawforAtAnsys I like it

**Comment by dcrawforAtAnsys** [comment]
To be addressed in PR #4019.

**Comment by gmalinve** [comment]
Closed by #4019.
